.travis.yml

```diff
@@ -0,0 +1,16 @@
+notifications:
+  slack: wtsi-cgpit:ptUMR1tkNyZJYd9TpGoss8WR
+  email: false
+
+language: c
+
+sudo: false
+
+services:
+  - docker
+
+install:
+ - docker build -t dockstore-cgpbigwig .
+
+script:
+ - docker images | grep -c dockstore-cgpbigwig
```

Comment:
As this is nice and contained it may be sensible to do a quick test of each tool, e.g.:

```
docker run -ti --rm IMAGENAME SCRIPT --version/--help
```

You don't need the tag it will just use what ever is available locally (I think).

Reply:
Added in https://github.com/cancerit/dockstore-cgpbigwig/pull/1/commits/755cc3bdb95b554bd05ab14e87b76cfc09dae7d3